templates/cookies/cookies_settings.twig

```diff
@@ -0,0 +1,47 @@
+{% set necessary_cookies_name = cookies.settings_copy.necessary_cookies_name %}
+{% set necessary_cookies_description = cookies.settings_copy.necessary_cookies_description %}
+{% set analytical_cookies_name = cookies.settings_copy.analytical_cookies_name %}
+{% set analytical_cookies_description = cookies.settings_copy.analytical_cookies_description %}
+{% set all_cookies_name = cookies.settings_copy.all_cookies_name %}
+{% set all_cookies_description = cookies.settings_copy.all_cookies_description %}
+
+<div class="d-none cookies-settings">
+	<div class="close-cookies-settings"></div>
+	<div class="cookies-settings-header">
+		<h4>{{__('Manage your cookies preferences', 'planet4-master-theme')}}</h4>
+		<p>{{__('Please select which cookies you are willing to store.', 'planet4-master-theme')}}</p>
+	</div>
+	<div class="cookies-settings-details">
```

Comment:
I'm testing with empty descriptions configuration (I assume it will be the case for some NROs when we deploy this, I'm testing on Africa DB which has only the general text set up) and it ends up looking a bit weird
![Screenshot from 2022-02-09 08-22-04](https://user-images.githubusercontent.com/617346/153141811-157dc48a-9e4e-494e-be57-b79a58f91851.png)
Do we have plan for this configuration ? We could add a condition in the template that checks for `{% if necessary_cookies_name is not empty or (cookies.enable_analytical_cookies and analytical_cookies_name is not empty) or all_cookies_name is not empty %}` to display the `cookies-settings-details` div and the sentence asking to select cookies, or maybe it would look better if we showed the "always enabled" cookie there, no idea :shrug:

Reply:
The plan for this was to run a script on all sites before release, to fill in these fields with the values that are currently in their respective Cookies blocks... But indeed just in case I'll add a check to make sure that we don't display anything settings-related in case there's nothing to show there!

Reply:
[Updated](https://github.com/greenpeace/planet4-master-theme/pull/1580/commits/d3d80cb96032bb48461ec5ba09e0483ae1788fd7) 🙂